src/com/palmergames/bukkit/towny/command/PlotCommand.java

```diff
@@ -400,8 +412,17 @@ public boolean parsePlotCommand(Player player, String[] split) throws TownyExcep
 									new PlotClaim(plugin, player, resident, selection, false, false, false).start();
 									continue;
 								}
+								
+								// Create confirmation.
+								Confirmation confirmation = new Confirmation(player);
+								List<WorldCoord> finalSelection = selection;

```

Comment:
This selection will only be one townblock and needs to be populated with the PlotGroup's townblocks like the group claiming confirmation above.